spec/spec_helper_acceptance.rb

```diff
@@ -1,33 +1,22 @@
-require 'json'
-require 'beaker-rspec/spec_helper'
-require 'beaker-rspec/helpers/serverspec'
+require 'beaker-rspec'
+require 'beaker-puppet'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
 
-unless ENV['BEAKER_provision'] == 'no'
-  hosts.each do |host|
-    # Install Puppet
-    if host.is_pe?
-      install_pe
-    else
-      install_puppet
-    end
-  end
-end
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_module
+install_module_dependencies
 
 RSpec.configure do |c|
-  # Project root
-  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
-
-  # Readable test descriptions
-  c.formatter = :documentation
-
   # Configure all nodes in nodeset
   c.before :suite do
-    # Install module and dependencies
-    puppet_module_install(source: proj_root, module_name: 'r10k')
-    hosts.each do |host|
-      JSON.parse(File.read('metadata.json'))['dependencies'].each do |dependency|
-        on host, puppet('module', 'install', dependency['name'].to_s), acceptable_exit_codes: [0, 1]
-      end
+    hosts.each do |_host|
+      ensure_puppet_user_exists = %(
+        user { 'puppet':
+          ensure => present,
+        }
+      )
+      apply_manifest(ensure_puppet_user_exists, catch_failures: true)
```

Comment:
A shorter way to write this is `on host, puppet('resource', 'user', 'ensure=present')`. Otherwise you should `apply_manifest_on(host, ensure_puppet_user_exists, catch_failures: true)` for it to work correctly on multiple nodes

Reply:
With this shorter way i got this log : 
```
centos7-64-1 23:05:44$ puppet resource user ensure=present
  user { 'ensure=present':
    ensure => 'absent',
  }
```
And even with `puppet resource user puppet`, it is yet produced `ensure=>absent`. hu!

Reply:
Oh, it should have been `on host, puppet('resource', 'user', 'puppet', 'ensure=present')`

Reply:
And you should have at least looked at my other comment. This current version is applying the same manifest only on the default node for every host. In a single node env that doesn't matter, but it's a bad example and people copy paste.